examples/step-60/step-60.cc

```diff
@@ -839,7 +850,7 @@ namespace Step60
     space_dh->distribute_dofs(*space_fe);
 
     DoFTools::make_hanging_node_constraints(*space_dh, constraints);
-    for (auto id : parameters.homogeneous_dirichlet_ids)
+    for (auto id : parameters.dirichlet_ids)
       {
         VectorTools::interpolate_boundary_values(
           *space_dh, id, Functions::ZeroFunction<spacedim>(), constraints);
```

Comment:
Shouldn't this be the `embedding_dirichlet_boundary_values` instead of `Functions::ZeroFunction<spacedim>()`?

Reply:
Thanks Luca for the review. 

> I think you are still missing a small change in the `assemble_system` function, since you also need to pass the `embedding_rhs` to the MatrixTools::create_laplace_matrix function

Here, `embedding_value_function` is related to the first component of the right-hand side, namely `embedding_rhs`, and both are used in the call to `MatrixTools::create_laplace_matrix()`.

Reply:
Ah, ok. In the comment to the PR you talked about Dirichlet BCs, but your changes actually add a non homogeneous rhs, rather than non-homogeneous Dirichlet BCs.

Reply:
If the user doesn't change `Functions::ZeroFunction<spacedim>()`, then you're right: I only allowed a possibly non-zero rhs.

But with my changes, if the "f"  in the original problem is 0, and the user specifies a non-homogeneous Dirichlet BCs on dOmega, the call to `create_laplace_matrix()` will distribute correctly the inhomogeneous constraints to the rhs vector. Before this PR, those constraints were not distributed in this scenario. Of course everything works with a non-zero "f" too, like in the picture above.